docs/source/markdown/podmansh.1.md

```diff
@@ -0,0 +1,111 @@
+% podmansh 1
+
+## NAME
+podmansh - Execute login shell within the Podman `podmansh` container
+
+## SYNOPSIS
+**podmansh**
+
+## DESCRIPTION
+
+Execute a user shell within a container when the user logs into the system. The container that the users get added to can be defined via a Podman Quadlet file. This user only has access to volumes and capabilities configured into the quadlet file.
+
+Administrators can create a quadlet in /etc/containers/systemd/users, which systemd will start for all users when they log in. The administrator can create a specific quadlet with the container name `podmansh`, then enable users to use the login shell /usr/bin/podmansh.  These user login shells are automatically executed inside  the `podmansh` container via Podman.	.
+
+Optionally, the administrator can place quadlet files in the /etc/containers/systemd/users/${UID} directory for a user. Only this UID will execute these quadlet services when that user logs in.
+
+The user is confined to the container environment via all of the security mechanisms, including SELinux. The only information that will be available from the system comes from volumes leaked into the container.
+
+Systemd will automatically create the container when the user session is started. Systemd will take down the container when all connections to the user session are removed. This means users can log in to the system multiple times, with each session connected to the same container.
+
+## Setup
+Modify user login session using usermod
+
+```
+# usermod -s /usr/bin/podmansh testu
+# grep testu /etc/passwd
+testu:x:4004:4004::/home/testu:/usr/bin/podmansh
```

Comment:
I think it's important to mention/create also /home/testu/data directory, podmansh container would not work without it

Reply:
Yes I think we got to continue to work on this man page. Although I would not hold up the PR.  Could you suggest how you want it stated. 

I still have a concern on whether or not we need to do `systemctl --user daemon-reload` after you place a quadlet file on disk, or does this happen automatically the first time a user logs in?

Reply:
The following changes in container file will solve it without need to document it:

```
--- a/docs/source/markdown/podmansh.1.md
+++ b/docs/source/markdown/podmansh.1.md
@@ -69,6 +69,9 @@ RunInit=yes
 Volume=%h/data:%h:Z
 Exec=sleep infinity
 
+[Service]
+ExecStartPre=/usr/bin/mkdir -p %h/data
+
 [Install]
 RequiredBy=default.target
 _EOF
@@ -99,6 +102,9 @@ Volume=%h/data:%h:Z
 Volume=/sys/fs/selinux:/sys/fs/selinux
 Exec=sleep infinity
 
+[Service]
+ExecStartPre=/usr/bin/mkdir -p %h/data
+
 [Install]
 RequiredBy=default.target
 _EOF
```

Reply:
> Or do we need to do sudo machinectl shell $USER /usr/bin/systemctl --user daemon-reload
> 
> For every user we want to use the podmansh quadlet.

No need to `daemon-reload` when there's no user session. systemd user session are created on first login session and closed when the last session ends so it happens automatically